### ISSUE TITLE
Release v1.0.0

### ISSUE THREAD

**Issue by xmfcx** [autowarefoundation/AWSIM-Labs#15]
## Blocked by

- https://github.com/autowarefoundation/AWSIM/issues/12
- https://github.com/autowarefoundation/AWSIM/issues/6
- https://github.com/autowarefoundation/AWSIM/issues/13
- https://github.com/autowarefoundation/AWSIM/issues/14
- https://github.com/autowarefoundation/AWSIM/issues/16
- https://github.com/autowarefoundation/AWSIM/issues/35
- ~https://github.com/autowarefoundation/AWSIM/issues/34~ Let's postpone this to the next sub release.
- https://github.com/autowarefoundation/AWSIM/issues/49
- https://github.com/autowarefoundation/AWSIM/issues/55
- https://github.com/autowarefoundation/AWSIM/pull/57

## Definition of done

- [x] Produce binaries for both scenario simulation and default AWSIM
- [x] Test them
- [x] Upload them
- [x] Update the documentation with these binaries

**Comment by lchojnack** [comment]
I've started to work on this task. During set up I've encountered the issue described [here](https://github.com/autowarefoundation/AWSIM/issues/49). I think that the issue blocks this task.

**Comment by xmfcx** [comment]
@lchojnack we have also merged the latest updates to the documentation with:
- https://github.com/autowarefoundation/AWSIM/pull/42


**Updated documentation page:** https://autowarefoundation.github.io/AWSIM/main/

We are now waiting for the release, feel free to ask if you have any questions for this task.

**Comment by lchojnack** [comment]
@xmfcx 
I've produced the AWSIM binaries and here are my observations

### AWSIM binary test
I've tested the binary with [Autoware on tag 2024.04](https://github.com/autowarefoundation/autoware/tree/2024.04). 

The binary works well, but I've noticed that green traffic lights are not detected well. 
I've compared the result with the HDRP version of AWSIM to check if it was caused by the rendering pipeline. I think that changing to URP reduces Autoware traffic light detection performance. 

**Question:** Is the not good green light detection acceptable for this release?

| URP (AWSIM Labs) | HDRP  ([AWSIM v.1.2.1](https://github.com/tier4/AWSIM/releases/tag/v1.2.1)) |
| --- | --- |
| <video src="https://github.com/autowarefoundation/AWSIM/assets/8416175/3d5f4225-c599-4f04-a2b0-db7562801eed" width="500" /> | <video src="https://github.com/autowarefoundation/AWSIM/assets/8416175/182430a6-1789-4f43-ab99-c3e90694a983" width="500" /> |


### AWSIM ss2 binary test

I've notice that the AWSIM labs branding are not applicable for ss2 scene.

**Question:** Is it acceptable for this release or just wait for fix?

| AWSIM bin | AWSIM ss2 bin |
| --- | --- |
| ![Screenshot_20240418_082037](https://github.com/autowarefoundation/AWSIM/assets/8416175/abe01572-7ae2-43b9-a967-0e0b597d973e) | ![Screenshot_20240418_082111](https://github.com/autowarefoundation/AWSIM/assets/8416175/ff43fb60-bc95-4204-8fba-cdb1cda0da0e) |


### Github release
**Questions:**
- Could you give me some more permission for [creating/manages releases for the repo](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository#about-release-management)? 


- To be sure, What version is this realese? 1.2.0?

**Comment by xmfcx** [comment]
Thanks @lchojnack 

## Traffic light problem

Before release, let's wait for this issue to be resolved:

- https://github.com/autowarefoundation/AWSIM/issues/55

## Scenario simulator

For this release, let's keep scenario simulator out, we can do it in the next release.

## Permissions

I've added you to the repository with Write permissions, please let me know if it doesn't work for you.

## Version

This will be the `1.0.0` for the Autoware Labs.

**Comment by lchojnack** [comment]
@xmfcx 

If the release will be `1.0.0`, the version in left-bottom corner of UI shouldn't also point to `1.0.0` instead `1.2.0`?

![Screenshot_20240418_082037](https://github.com/autowarefoundation/AWSIM/assets/8416175/2d05a570-8fc8-43ee-a0e1-283b456f6ff2)

**Comment by xmfcx** [comment]
Yes, It should be a part of the release PR, along with the doc updates.

**Comment by xmfcx** [comment]
I will release the current state with 3 lidars as a beta version `1.0.0-beta` along with
- https://github.com/autowarefoundation/autoware/issues/4659

To update relevant readme files and instructions.

When
- https://github.com/autowarefoundation/AWSIM/pull/57

is merged, let's to the 1.0.0 release.

**Comment by lchojnack** [comment]
@mozhoku @xmfcx 
I'm trying to release v1.0.0, and I've noticed that the Graphy is missing when I run the AWISM from the main. The Graphy in the videos from https://github.com/autowarefoundation/AWSIM/pull/57 is present.

![image](https://github.com/autowarefoundation/AWSIM/assets/8416175/d6379d8d-cdfc-41cd-a2bd-15c976b7d0d2)

I assume that the Graphy should be visible, and it is a bug on main branch or in my setup.

Could you confirm that it is a bug and Graphy should be visible in the new UI or it is as expected and Graphy shouldn't be visible?

**Comment by lchojnack** [comment]
@mozhoku @xmfcx 
Ok, I see what could be wrong. The Graphy is disabled on the scene by default.
Please let me know, Is it expected or is it a bug?

![image](https://github.com/autowarefoundation/AWSIM/assets/8416175/ce91760c-8cdb-45d4-b628-a638bad430d6)

**Comment by mozhoku** [comment]
@lchojnack 
It should be enabled by default. I've forgot to turn it on at some point, my bad. Could you please turn it on? :pray:

**Comment by lchojnack** [comment]
@mozhoku 
Thank you for the clarification. Yes, I will enable it.

**Comment by lchojnack** [comment]
@xmfcx 
Release v1.0.0 is ready https://github.com/autowarefoundation/AWSIM/releases/tag/v1.0.0

**Comment by xmfcx** [comment]
@brkay54 could you test this v1.0.0 binary against the latest main branch of the Autoware?

**Comment by xmfcx** [comment]
Test video with AWSIM Labs 1.0.0: https://www.youtube.com/watch?v=GPow8YPgi1s

Thanks!
